client/js/modalContent.js

```diff
@@ -6,15 +6,14 @@ const getTitleAndDateFromModal = async() => {
     */
 
     // Reads the title name of movie / series from the previewModal--player-titleTreatment-logo class
-    let titleElement = document.querySelectorAll('.previewModal--player-titleTreatment-logo')
-    let title = (titleElement === undefined  || titleElement[0] === undefined ) ? null : titleElement[0].getAttribute("alt")
-
+    let titleElement = document.getElementsByClassName('previewModal--player-titleTreatment-logo')
+    let title = (titleElement === undefined) ? null : (titleElement[0] === undefined ? null : titleElement[0].getAttribute("alt"))
```

Comment:
we can just say

let title = titleElement ? null : titleElement[0] ? null : titleElement[0].getAttribute('alt')

== undefined is not needed right? undefined and null are falsy